package.json

```diff
@@ -41,7 +41,7 @@
     "react-dates": "^21.8.0",
     "react-dom": "^16.13.1",
```

Comment:
Are these updated by design?

Reply:
The branch commit was messed up.
I've fixed it, it required a push -f in this specific case.